### PR TITLE
Parameter order must be retained for positional bindings to work. Replace HashMap to LinkedHashMap to retain the order.

### PR DESCRIPTION
**Pull request by cthumuluru**

### CODE REVIEW

File: common/src/main/java/com/datastax/oss/simulacron/common/stubbing/PrimeDsl.java [hunk 1]
```diff
@@ -162,7 +162,7 @@ public static SuccessResult noRows() {
    * @return A rows response instance.
    */
   public static SuccessResult rows(
-      List<Map<String, Object>> rows, Map<String, String> columnTypes) {
+      List<LinkedHashMap<String, Object>> rows, LinkedHashMap<String, String> columnTypes) {
```
Comment (jorgebay) [inline common/src/main/java/com/datastax/oss/simulacron/common/stubbing/PrimeDsl.java:165]: Makes sense to also use a `LinkedHashMap` for row cell values 👍

**Comment by jorgebay** [comment]
@joao-r-reis @adutra @emerkle826 Do you want (and have time to) review it before merging it?

**Comment by adutra** [comment]
I don't have time for a proper review, sorry, but the changes look good to me. The only nit is that we don't have any test that exercises parameter order, it would be nice if we could validate the changes introduced here by such a test. But feel free to merge nonetheless.

**Comment by cthumuluru** [comment]
Hey, thanks for the review. I can look into adding the tests.

**Comment by cthumuluru** [comment]
Added test to verify valid & invalid positional param bindings. Also included a test for named parameter bindings.

**Comment by adutra** [comment]
> Added test to verify valid & invalid positional param bindings. Also included a test for named parameter bindings.

Absolutely awesome, thanks! I will fix a few nits and commit to master shortly.

**Comment by adutra** [comment]
Committed to master as 0d1fb5c. Thanks!

**Comment by cthumuluru** [comment]
@adutra — Thank you. This is a great mocking tool you guys have built. :)

**Pull request closed without merging**